algorithm0.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+#include "common.h"
+#include "algorithm0.h"
+
+
+int algorithm0(char marker, char* copyboard){
```

Comment:
switch the parameters of all algorithms according PlayersAlgorithms.h in main branch